api/Headers.json

```diff
@@ -4,85 +4,30 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Headers",
         "support": {
-          "chrome": [
-            {
-              "version_added": "42"
-            },
-            {
-              "version_added": "41",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Experimental Web Platform Features"
-                }
-              ]
-            }
-          ],
-          "chrome_android": [
-            {
-              "version_added": "42"
-            },
-            {
-              "version_added": "41",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Experimental Web Platform Features"
-                }
-              ]
-            }
-          ],
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
           "edge": {
             "version_added": "14"
           },
-          "firefox": [
-            {
-              "version_added": "39"
-            },
-            {
-              "version_added": "34",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.fetch.enabled"
-                }
-              ]
-            }
-          ],
+          "firefox": {
+            "version_added": "39"
```

Comment:
I doubt this data is correct, that `fetch()` would have shipped earlier in desktop than on Android. This change makes the difference more obvious. Are you able to test for when this was shipped in Firefox for Android?

Looks like the data came from wiki migration: https://github.com/mdn/browser-compat-data/pull/929

Reply:
I'll have to see if I can run a Firefox Android instance that old, but for now, I've opened up #9880.